tesseract_collision/core/include/tesseract_collision/core/types.h

```diff
@@ -239,6 +240,20 @@ enum class CollisionEvaluatorType
   LVS_CONTINUOUS
 };
 
+/** @brief Identifies how the provided AllowedCollisionMatrix should be applied relative to the isAllowedFn in the
+ * contact manager */
+enum class ACMOverrideType
+{
+  /** @brief Do not apply AllowedCollisionMatrix */
+  NONE,
+  /** @brief Replace the current IsContactAllowedFn with one generated from the ACM provided */
+  ASSIGN,
+  /** @brief New IsContactAllowedFn combines the contact manager fn and the ACM generated fn with and AND */
+  AND,
```

Comment:
This is interesting. Did you have a use case in mind?

Reply:
Not specifically. Maybe you want to do a collision check of 2 arms and only ignore pairs in the acm for both manipulators?